CHANGELOG.md

```diff
@@ -3,6 +3,78 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.15.1
+_Sep 28, 2023_
+
+We'd like to offer a big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- 🧰 Fix header filters menu auto closing on render (#10483) @MBilalShafi
+- 🌳 Do not try to focus a collapsed node when re-focusing the TreeView (#10422) @flaviendelangle
```

Comment:
```suggestion
- 🌳 Fix focusing of a collapsed node when the TreeView is re-focused (#10422) @flaviendelangle
```

Reply:
I don't think this is worth highlighting, it's a minor bug fix

You can replace it with the locales, we always highlight them

Reply:
Also, a nitpick: Have these emojis ever been used for highlights? 🤔 
WDYT about sticking to what we already had "on rotation"? 🤔

Reply:
I agree that consistency is important here 👍 But I kind of like having a tree emoji for treeview features 🤔

Reply:
Fair point. 👌 
However, if we agree with Flavien and remove it from the highlights, we can keep this emoji for later highlights. 👍

Reply:
Sure! I was just pointing out that I like the little tree. Let's remove this particular highlight and keep that in mind for the future

Reply:
> Also, a nitpick: Have these emojis ever been used for highlights? 🤔 WDYT about sticking to what we already had "on rotation"? 🤔

Oh, I was under the impression that the docs, l10n and bugfixes all have assigned emojis, but the rest are up for what fits best! :D

Reply:
I'm not sure if there are "assigned" emojis, just wanted to signal that we usually use a bit different ones. 👌 
It totally makes sense to add a nice emoji if it makes sense (like the 🌳  for TreeView and a 🧹  for the clearable behavior). 🙌